packages/apps/sale/src/screens/Cart/ActiveCartScreen.tsx

```diff
@@ -0,0 +1,205 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2024 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useCallback, useEffect, useMemo, useState} from 'react';
+import {
+  Button,
+  DoubleIcon,
+  Label,
+  LabelText,
+  Screen,
+  useThemeColor,
+} from '@axelor/aos-mobile-ui';
+import {
+  headerActionsProvider,
+  SearchListView,
+  useDispatch,
+  useSelector,
+  useTranslator,
+} from '@axelor/aos-mobile-core';
+import {searchCart, searchCartLine, updateCart} from '../../features/cartSlice';
+import {CartLineActionCard, CustomerSearchBar} from '../../components';
+import {StyleSheet} from 'react-native';
+
+const ActiveCartScreen = ({}) => {
+  const dispatch = useDispatch();
+  const Colors = useThemeColor();
+  const I18n = useTranslator();
+
+  const {mobileSettings} = useSelector((state: any) => state.appConfig);
+  const {loading, moreLoading, isListEnd, cartList, carLineList} = useSelector(
+    (state: any) => state.sale_cart,
+  );
+  const {userId} = useSelector((state: any) => state.auth);
+
+  const [selectedCustomer, setSelectedCustomer] = useState(null);
+
+  useEffect(() => {
+    dispatch((searchCart as any)({userId}));
+  }, [dispatch, userId]);
+
+  const activeCart = useMemo(() => {
+    return cartList[0];
+  }, [cartList]);
+
+  useEffect(() => {
+    setSelectedCustomer(activeCart?.partner);
+  }, [activeCart?.partner]);
+
+  useEffect(() => {
+    if (activeCart) {
+      headerActionsProvider.registerModel('sale_active_cart', {
+        model: 'com.axelor.apps.sale.db.Cart',
+        modelId: activeCart?.id,
+        disableMailMessages: !mobileSettings?.isTrackerMessageEnabled,
+        attachedFileScreenTitle: activeCart?.name,
```

Comment:
name does not exist on cart object... 
Remove this option, it won't be useful here